Xkcd-Comics/ContentViews/ComicsView.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  ContentView.swift
+//  Xkcd-Comics
+//
+//  Created by Jia-Jiuan Tsai on 2021-11-24.
+//
+
+import SwiftUI
+import Kingfisher
+
+struct ComicsView: View {
+    @ObservedObject var viewModel: ComicsViewModel
+    
+    var body: some View {
+        NavigationView {
+            VStack {
+                Spacer()
+                ComicView(viewModel: viewModel)
```

Comment:
It's great that you divided the view up like this. We can now much more easily understand the structure and find specific parts that we are looking for, without first having to understand all of the details of all parts.